library/php

```diff
@@ -41,17 +41,17 @@ apache: git://github.com/docker-library/php@cf1e938f3721632443e01734bcfcbcf1160e
 5-fpm: git://github.com/docker-library/php@cf1e938f3721632443e01734bcfcbcf1160ea539 5.6/fpm
 fpm: git://github.com/docker-library/php@cf1e938f3721632443e01734bcfcbcf1160ea539 5.6/fpm
 
-7.0.0RC3-cli: git://github.com/docker-library/php@5b0e536d2007e886f65856f319dc36fd5c011c79 7.0
-7.0-cli: git://github.com/docker-library/php@5b0e536d2007e886f65856f319dc36fd5c011c79 7.0
-7-cli: git://github.com/docker-library/php@5b0e536d2007e886f65856f319dc36fd5c011c79 7.0
-7.0.0RC3: git://github.com/docker-library/php@5b0e536d2007e886f65856f319dc36fd5c011c79 7.0
-7.0: git://github.com/docker-library/php@5b0e536d2007e886f65856f319dc36fd5c011c79 7.0
-7: git://github.com/docker-library/php@5b0e536d2007e886f65856f319dc36fd5c011c79 7.0
-
-7.0.0RC3-apache: git://github.com/docker-library/php@5b0e536d2007e886f65856f319dc36fd5c011c79 7.0/apache
-7.0-apache: git://github.com/docker-library/php@5b0e536d2007e886f65856f319dc36fd5c011c79 7.0/apache
-7-apache: git://github.com/docker-library/php@5b0e536d2007e886f65856f319dc36fd5c011c79 7.0/apache
-
-7.0.0RC3-fpm: git://github.com/docker-library/php@5b0e536d2007e886f65856f319dc36fd5c011c79 7.0/fpm
-7.0-fpm: git://github.com/docker-library/php@5b0e536d2007e886f65856f319dc36fd5c011c79 7.0/fpm
-7-fpm: git://github.com/docker-library/php@5b0e536d2007e886f65856f319dc36fd5c011c79 7.0/fpm
+7.0.0RC3-cli: git://github.com/docker-library/php@9cfb9ff18f98297b00f0ad90af6c13692411649f 7.0
+7.0-cli: git://github.com/docker-library/php@9cfb9ff18f98297b00f0ad90af6c13692411649f 7.0
+7-cli: git://github.com/docker-library/php@9cfb9ff18f98297b00f0ad90af6c13692411649f 7.0
+7.0.0RC3: git://github.com/docker-library/php@9cfb9ff18f98297b00f0ad90af6c13692411649f 7.0
+7.0: git://github.com/docker-library/php@9cfb9ff18f98297b00f0ad90af6c13692411649f 7.0
+7: git://github.com/docker-library/php@9cfb9ff18f98297b00f0ad90af6c13692411649f 7.0
+
+7.0.0RC3-apache: git://github.com/docker-library/php@9cfb9ff18f98297b00f0ad90af6c13692411649f 7.0/apache
+7.0-apache: git://github.com/docker-library/php@9cfb9ff18f98297b00f0ad90af6c13692411649f 7.0/apache
+7-apache: git://github.com/docker-library/php@9cfb9ff18f98297b00f0ad90af6c13692411649f 7.0/apache
+
+7.0.0RC3-fpm: git://github.com/docker-library/php@9cfb9ff18f98297b00f0ad90af6c13692411649f 7.0/fpm
```

Comment:
It still has RC3 :open_mouth:, but the commit id is correct. If you have the php repo checked out, just run the `generate-stackbrew-library.sh` and direct the ouput to this file :+1:.